pkgs/tools/networking/mtr-exporter/default.nix

```diff
@@ -0,0 +1,27 @@
+{ pkgs ? import <nixpkgs> { } }:
+
+pkgs.buildGoModule rec {
+  pname = "mtr-exporter";
+  version = "2020-03-08";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "mgumz";
+    repo = "mtr-exporter";
+    rev = "ab0b97a987c000ebf526691a6a84fbc5890bd632";
+    sha256 = "0715az97ipn36b35h6ys1jqhlcl8w6w0qzg7rnbgj4fraip2nfdg";
+  };
+
+  vendorSha256 = "0njn0ac7j3lv8ax6jc3bg3hh96a42jal212qk6zxrd46nb5l1rj8";
+
+  patches = [ ./fix-handle-MTR-JSON-format-changes.patch ];
```

Comment:
since it's already posted in a pull request you could also get the patch from there using `fetchurl`.